Frontend/app/user/signin/signin.html

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <div class="col-xs-12">
+    <div class="welcome">
+      <h1 class="text-brand">
+        mentii<br/>
+        <hr style="width: 50%;">
```

Comment:
I was always told to avoid style in the html, so if we have a general css for this page shove that over there, if not don't make a whole new one for this ONE style on this page lol.